src/gui/static/e2e/wallets.po.ts

```diff
@@ -0,0 +1,231 @@
+import { browser, by, element, protractor } from 'protractor';
+
+export class WalletsPage {
+  navigateTo() {
+    return browser.get('/#/wallets');
+  }
+
+  getHeaderText() {
+    return element(by.css('.title')).getText();
+  }
+
+  showAddWallet() {
+    const btnAdd = element(by.buttonText('Add Wallet'));
+
+    return btnAdd.click().then(() => {
+      return element(by.css('app-create-wallet')).isPresent();
+    });
+  }
+
+  showLoadWallet() {
+    const btnLoad = element(by.buttonText('Load Wallet'));
+
+    return btnLoad.click().then(() => {
+      return element(by.css('app-create-wallet')).isPresent();
+    });
+  }
+
+  getWalletModalTitle() {
+    return element(by.css('app-create-wallet .-header')).getText();
+  }
+
+  fillWalletForm(label: string, seed: string, confirm: string|null) {
+    const labelEl = element(by.css('[formcontrolname="label"]'));
+    const seedEl = element(by.css('[formcontrolname="seed"]'));
+    const btn = element(by.buttonText(confirm ? 'Create' : 'Load'));
+
+    labelEl.clear();
+    seedEl.clear();
+    labelEl.sendKeys(label);
+    seedEl.sendKeys(seed);
+
+    if (confirm) {
+      const confirmEl = element(by.css('[formcontrolname="confirm_seed"]'));
+      confirmEl.clear();
+      confirmEl.sendKeys(seed);
+    }
+
+    return btn.isEnabled();
+  }
+
+  createWallet() {
```

Comment:
If `fillWalletForm` is modified a bit, it is possible to use that function to fill in the form, thus avoiding to repeat the code of filling the form in this function and others. These modifications on `fillWalletForm` may be needed anyways to solve the problem with `should validate create wallet, seed mismatch`.